lib/validation.tsx

```diff
@@ -349,8 +349,9 @@ export const isNumber = (field: string): boolean => {
  * @param field A string containing a symbol character
  * @returns {boolean} The validation result
  */
-export const isSymbol = (field: string): boolean => {
-  const reg = new RegExp("^(?=.*?[#?!@$%^&*-])");
+export const containsSymbol = (field: string): boolean => {
+  // eslint-disable-next-line no-useless-escape
+  const reg = /^(?=.*?[\^\$\*\.\[\]\{\}\(\)\?\"\!\@\#\%\&\/\\\,\>\<\'\:\;\|\_\~\`\=\+\-])/;
```

Comment:
The regex looks like it only matches symbols and I think you got all of them :) If you have a reference for where you got this regex it might help, then again maybe you wrote it from scratch

Reply:
I got mine from here that I was using as a base.  https://stackoverflow.com/questions/58767980/aws-cognito-password-regex-specific-to-aws-cognito

Reply:
It does only look for symbols. Then we have other regex to validate other password requirements ;) . I wrote it myself using the set of special characters listed [here](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-policies.html).